modules/3d-tiles/src/parsers/parse-3d-tile-point-cloud.js

```diff
@@ -154,12 +156,25 @@ function parseNormals(tile, featureTable) {
     if (featureTable.hasProperty('NORMAL')) {
       tile.attributes.normals = featureTable.getPropertyArray('NORMAL', GL.FLOAT, 3);
     } else if (featureTable.hasProperty('NORMAL_OCT16P')) {
-      tile.attributes.normals = featureTable.getPropertyArray('NORMAL_OCT16P', GL.UNSIGNED_BYTE, 2);
+      const normals = featureTable.getPropertyArray('NORMAL_OCT16P', GL.UNSIGNED_BYTE, 2);
       tile.isOctEncoded16P = true;
+      decodeOctNormals(normals, tile);
     }
   }
 }
 
+function decodeOctNormals(normals, tile) {
```

Comment:
Should we put this function in the same helper file where we have the RGB565 color processing? Maybe rename that file to `normalize-attributes.js` ...